File: C++/Lista extra 02/2.c
#include <stdio.h>

int main (){
    float n1;

    printf("Informe um numero:\n");
    scanf("%f",&n1);

    if(n1>0){
        printf("%f e positivo\n", n1);
    }
    if(n1<0){
        printf("%f e negativo\n", n1);
    }

    return 0;
}

File: PYTHON/LISTA 2/2.py
#Faça um Programa que peça um valor e mostre na tela se o valor é positivo ou negativo.
try:
    num = float(input("Informe um numero: "))

    if num > 0:
        print("{} é positivo".format(num))
        
    elif num < 0:
        print("{} é negativo".format(num))
        
    elif num == 0:
        print("{} é neutro".format(num))

except ValueError:
    print("Por favor, digite um número válido.")